ibtk/src/boundary/physical_boundary/CartExtrapPhysBdryOp.cpp

```diff
@@ -375,13 +359,11 @@ CartExtrapPhysBdryOp::setPhysicalBoundaryConditions_cell(
         const Box<NDIM>& ghost_box = patch_data->getGhostBox();
 
         // Loop over the boundary fill boxes and extrapolate the data.
-        for (std::vector<std::pair<Box<NDIM>, std::pair<int, int> > >::const_iterator it = bdry_fill_boxes.begin();
-             it != bdry_fill_boxes.end();
-             ++it)
+        for (const auto& bdry_fill_box_it : bdry_fill_boxes)
```

Comment:
I am not sure how I feel about the use of `XXX_it` for variables that are no longer iterators...

Reply:
This could be a future PR, but need to change `bdry_fill_box_it` to something else, since the type is not an interator type.